x-pack/plugins/apm/public/components/shared/service_link.tsx

```diff
@@ -35,22 +53,92 @@ export function ServiceLink({
     ? '/mobile-services/{serviceName}/overview'
     : '/services/{serviceName}/overview';
 
+  if (serviceName === serviceDroppedBucketName) {
+    return (
+      <>
+        <EuiToolTip
+          content={i18n.translate('xpack.apm.serviceLink.tooltip.message', {
+            defaultMessage:
+              "The maximum number of services were reached. Please see the APM Server docs for 'aggregation.service.max_groups' to increase this",
+          })}
+        >
+          <EuiFlexGroup alignItems="center" gutterSize="xs">
+            <EuiFlexItem>
+              {i18n.translate('xpack.apm.serviceLink.other.label', {
+                defaultMessage: '_other',
+              })}
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiIcon size="s" color="subdued" type="alert" />
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiToolTip>
+
+        <EuiFlexGroup alignItems="center" gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            {i18n.translate('xpack.apm.serviceLink.other.label', {
+              defaultMessage: '_other',
+            })}
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <PopoverTooltip
+              ariaLabel={i18n.translate('xpack.apm.serviceLink.tooltip', {
+                defaultMessage: 'Max service groups reached tooltip',
+              })}
+              iconType="alert"
+            >
+              <EuiText style={{ width: `${unit * 28}px` }} size="s">
+                <FormattedMessage
+                  defaultMessage="The service group limit has been reached. Please see the {apmServerDocs} for 'aggregation.service.max_groups' to increase this."
+                  id="xpack.apm.serviceLink.tooltip.message"
```

Comment:
Let's not use the term "service groups" since it means something different :)

```suggestion
                  defaultMessage="The maximum number of unique services has been reached. Please see the {apmServerDocs} for 'aggregation.service.max_groups' to increase this."
                  id="xpack.apm.serviceLink.tooltip.message"
```